pep-0480.txt

```diff
@@ -100,7 +107,9 @@ Threat Model
 
 The threat model assumes the following:
 
-* Offline keys are safe and securely stored.
+* Developer keys are either offline or ephemeral. Offline keys are safe and
```

Comment:
This is a fairly strong assumption.  Surely at least some such keys will be able to be controlled by an attacker.  We don't want all projects on the repo to be insecure if one project has its keys compromised.

Reply:
"Offline keys are safe and securely stored" was in the PEP when i first read it.
I added the option for ephemeral keys, which should be only written to RAM and never to disk and then flushed after use, usually within a couple minutes of creation.
Is your concern with the overall assumption (offline + ephemeral keys are safe) or the ephemeral part specifically?